services/filemanager/filemanager.go

```diff
@@ -133,7 +133,11 @@ func GetProviderConfigForBackupsFromEnv(ctx context.Context) map[string]interfac
 	return providerConfig
 }
 
-func getBatchRouterDurationConfig(key, destType string, defaultValueInTimescaleUnits int64, timeScale time.Duration) time.Duration {
+func getBatchRouterTimeoutConfig(destType string) time.Duration {
```

Comment:
I agree with this change, since I have proposed the same thing in the past, however I remember @chandumlg having a different opinion on that